app/modules/ads/index.js

```diff
@@ -413,6 +413,22 @@ class Ads {
     }
   }
 
+  /**
+   * @private
+   */
+  trackKruxSegments() {
+    const { context, utils } = window.Wikia.adEngine;
+    const kruxUserSegments = context.get('targeting.ksg') || [];
+    const kruxICBMSegments = context.get('services.krux.icbmSegments') || [];
+
+    const kruxPropValue = kruxUserSegments.filter(segment => kruxICBMSegments.includes(segment));
+
+    if (kruxPropValue.length) {
```

Comment:
AC: `empty string if in no segments from list`. Something changed?

Reply:
same conclusion as for desktop